pkgs/development/python-modules/sqliteschema/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mbstrdecoder
+, tabledata
+, typepy
+}:
+
+buildPythonPackage rec {
+  pname = "sqliteschema";
+  version = "1.4.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "thombashi";
+    repo = "sqliteschema";
+    rev = "v${version}";
+    hash = "sha256-IzHdYBnh6udVsanWTPSsX4p4PG934YCdzs9Ow/NW86E=";
+  };
+
+  propagatedBuildInputs = [
+    mbstrdecoder
+    tabledata
+    typepy
+  ];
+
+  pythonImportsCheck = [ "sqliteschema" ];
+  doCheck = false;
```

Comment:
Why not run the tests here https://github.com/thombashi/sqliteschema/tree/master/test ?

Reply:
Added a comment [here](https://github.com/NixOS/nixpkgs/blob/d0533394380c21ab7a09918729e7297268944ab7/pkgs/development/python-modules/sqliteschema/default.nix#L31-L32) to explain why the tests are not run. The reason is, enabling tests would trigger infinite recursion due to circular dependency between `sqliteschema` and `simplesqlite`.

Reply:
we can move them to passthru.tests to avoid this

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/299429/commits/9848b1931a132f54c8d1635879b1c50265045ed0